ansible/templates/cluster-template.yaml.j2

```diff
@@ -935,8 +1016,57 @@ etcd:
 #      content: |
 #        { "rktKind": "dockerAuth", "rktVersion": "v1", ... }
 #
+  customFiles:
+# We started dropping packets because the netfilter conntrack table filled up.
+# More info - https://trello.com/c/kuI6YJSv/747-packet-drop-investigation
+# 
+# The following changes:
```

Comment:
Can you add the previous values here so we get an idea about the scale of the changes in case we need to change them again?

Reply:
Yup, good idea.